### PR TITLE
new srt instance for exui

### PR DESCRIPTION
**Pull request by kaanaktas**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CCD-2552

### Change description ###
Deploy EUI into demo-srt

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by timja — APPROVED** [review]

File: apps/xui/xui-webapp-srt-integration/demo-image-policy.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: image.toolkit.fluxcd.io/v1alpha2
+kind: ImagePolicy
+metadata:
+  name: demo-xui-webapp-srt-integration
+  annotations:
+    hmcts.github.com/prod-automated: disabled
+spec:
+  filterTags:
+    pattern: '^pr-1811-[a-f0-9]+-(?P<ts>[0-9]+)'
```
Comment (MSancaktutar) [inline apps/xui/xui-webapp-srt-integration/demo-image-policy.yaml:9]: is this the particular EUI image currently in "the demo"?

File: apps/xui/automation/kustomization.yaml [hunk 2]
```diff
@@ -22,6 +22,9 @@ resources:
   - ../xui-webapp-wa-integration/image-repo.yaml
   - ../xui-webapp-wa-integration/demo-image-policy.yaml
   - ../xui-webapp-wa-integration/image-policy.yaml
+  - ../xui-webapp-srt-integration/image-repo.yaml
+  - ../xui-webapp-srt-integration/demo-image-policy.yaml
+  - ../xui-webapp-srt-integration/image-policy.yaml
```
Comment (MSancaktutar) [inline apps/xui/automation/kustomization.yaml:27]: `srt-integration` --> just `srt` please.

**Review by MSancaktutar — APPROVED** [review]
LGTM - we may tweak to fix forward, if needed.

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by cakeben — APPROVED** [review]

**Pull request merged**